### PR TITLE
a post about the new AWS FSx managed service

### PR DESCRIPTION
**Pull request by mbonig**
In a comment on the pull request, include the following information and check off the list.

 - [x] Mini bio
  - Soft capped to 75 words
 - [x] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [x] Social media contact info
 - [yes] Redaction: (yes|no)
  - Allow redaction, editing or structure changes by technical writer staff
 - [yes] Comments: (yes|no)
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-7b30e6d0dd/)

**Comment by catlook** [comment]
got it!!  will review today :)

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+published: false
+authorIsRacker: true
+authorAvatar: https://s.gravatar.com/avatar/17fefeb3c1832175bf6fbe9841368292?s=128
+bio: "As a Senior Software Engineer at Rackspace, Matthew draws upon his 15 years of web application development experience to help architect highly-available, fault-tolerant, scalable, and secure AWS environments composed of a wide range of services in the AWS portfolio, including Compute, Storage, Database, Networking, Developer Tools, and more. He is an AWS certified Solutions Architect. His hobbies include hiking the foothills of Colorado and walks with his wife and dogs."
+categories:
+  - aws
+  - Developers
+---
+
+Windows File Server have been one of the longest running methods for sharing files in corporate america for decades. Amazon brings its expertise in Serverless to provide a solution for the problem of the large Total Cost of Ownership that on-premise options have had in the past.
```
Comment (catlook) [inline _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: Using Windows File Servers has been ...
... in **serverless** solutions to solve the problem...

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## History
+
+Since before Microsoft Windows 95 graced the hearts of many, corporate America was using file shares to allow companies of all sizes manage the files their various business organizations needed. Early on these were largely Word and Excel documents but expanded to a wide variety of applications over the years. File shares were an easy system to setup and everybody could access the files they needed. Just setup an H: drive on your Microsoft Windows workstation and you were off and running. There was no specialized application, no Slack or Office365 or Google Drive. Just use Windows Explorer.
```
Comment (catlook) [inline _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: ... workstation, and you  ...

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+I'm setup file shares in my home, my parent's home, college, small businesses and used them in any company with a Windows laptop.
```
Comment (catlook) [inline _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: I've set up file shares ...
... college, and small businesses  ...

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+However, while file shares are easy to setup, there is a lot to manage. You need to manage the hardware, like provisioning new drives, setting them up, adding them to pools. Software maintenance takes the form of regular patches to Windows, ensuring proper licensing, and scheduling regular backups.
+
+```arnold
+I'm old, not obsolete.
+- Terminator, Terminator Genisys
+```
+
+## Amazon FSx
+
+**Amazon FSx for Windows File Server** is Windows File Server on Amazon's world-class serverless infrastructure. Just pick your total size and your throughput and you now have a DFS compatible Windows File Share system accessible from anywhere within the associated VPC.  There's no hardware to manage, no software to manage, you just pick your values and go.
```
Comment (catlook) [inline _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: ... throughput, and you now ...
... associated Virtual Private Cloud (VPC.)  There's ...

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+The system is 100% compatible with SMB 3.1.1 (earlier is supported but not recommended) and is a drop-in replacement for your existing Windows File Server. This is because AWS made a conscience decision to not re-implement any APIs, they're spinning up Windows Servers for you and managing everything. Another full managed-service offering.
```
Comment (catlook) [inline _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: ... a conscious decision ...
... APIs, so they're  ...

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+Authentication is supported using an AWS Active Directory service. You can use an AWS managed AD service. AD Connector support is coming soon.
```
Comment (catlook) [inline _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: ... Active Directory (AD) service ...

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+By default, your shares are **not** Multi-AZ. So, if high availability is a must, you can use DFS Namespaces across a few FSx instances. In fact, using Namespaces could get you 3 EB (3 million TBs) of storage (64TB per 50,000 shares).
+
+Currently this is only accessibly from within the associated VPCs, so machines on-prem or your laptops and workstations don't have access yet. CloudFormation templates also don't exist yet, so management is via the Console or through client API's like Python, Node, or others.  We should see soon CloudFormation support, AD Connector support, and VPN support, resolving many of the initial limitations of FSx.
```
Comment (catlook) [inline _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: ... Currently, this ...
... only accessible from ...
.. on-prem ()or your laptops and workstations) don't ...

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+The system is 100% compatible with SMB 3.1.1 (earlier is supported but not recommended) and is a drop-in replacement for your existing Windows File Server. This is because AWS made a conscience decision to not re-implement any APIs, they're spinning up Windows Servers for you and managing everything. Another full managed-service offering.
+
+Authentication is supported using an AWS Active Directory service. You can use an AWS managed AD service. AD Connector support is coming soon.
+
+By default, your shares are **not** Multi-AZ. So, if high availability is a must, you can use DFS Namespaces across a few FSx instances. In fact, using Namespaces could get you 3 EB (3 million TBs) of storage (64TB per 50,000 shares).
+
+Currently this is only accessibly from within the associated VPCs, so machines on-prem or your laptops and workstations don't have access yet. CloudFormation templates also don't exist yet, so management is via the Console or through client API's like Python, Node, or others.  We should see soon CloudFormation support, AD Connector support, and VPN support, resolving many of the initial limitations of FSx. 
+
+## Conclussion
```
Comment (catlook) [inline _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: ### Conclusion

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+authorAvatar: https://s.gravatar.com/avatar/17fefeb3c1832175bf6fbe9841368292?s=128
+bio: "As a Senior Software Engineer at Rackspace, Matthew draws upon his 15 years of web application development experience to help architect highly-available, fault-tolerant, scalable, and secure AWS environments composed of a wide range of services in the AWS portfolio, including Compute, Storage, Database, Networking, Developer Tools, and more. He is an AWS certified Solutions Architect. His hobbies include hiking the foothills of Colorado and walks with his wife and dogs."
+categories:
+  - aws
+  - Developers
+---
+
+Windows File Server have been one of the longest running methods for sharing files in corporate america for decades. Amazon brings its expertise in Serverless to provide a solution for the problem of the large Total Cost of Ownership that on-premise options have had in the past.
+
+## History
```
Comment (catlook) [inline _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: ### History

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+
+Authentication is supported using an AWS Active Directory service. You can use an AWS managed AD service. AD Connector support is coming soon.
+
+By default, your shares are **not** Multi-AZ. So, if high availability is a must, you can use DFS Namespaces across a few FSx instances. In fact, using Namespaces could get you 3 EB (3 million TBs) of storage (64TB per 50,000 shares).
+
+Currently this is only accessibly from within the associated VPCs, so machines on-prem or your laptops and workstations don't have access yet. CloudFormation templates also don't exist yet, so management is via the Console or through client API's like Python, Node, or others.  We should see soon CloudFormation support, AD Connector support, and VPN support, resolving many of the initial limitations of FSx. 
+
+## Conclussion
+
+Technologies that are built well tend to last. File Share did exactly what we needed, at the time, and still serves a powerfull role in corporate networks.
```
Comment (catlook) [inline _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: ... time and still ...

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md (hunk 9 above)
Comment (mbonig) [reply _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: Not sure I understand this edit, @catlook

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md (hunk 10 above)
Comment (mbonig) [reply _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: ... exactly what we needed at the time, and still services ... ??? @catlook

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-00483d573f/).

**Comment by mbonig** [comment]
> If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-00483d573f/).

Tried the link but don't see anything about this post...

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md (hunk 10 above)
Comment (catlook) [reply _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: don't need comma before and.  I was just removing the comma

File: _posts/2019-01-29-amazon-fsx-replaces-file-shares.md (hunk 9 above)
Comment (catlook) [reply _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: wanted ### instead of ##

Comment (mbonig) [reply _posts/2019-01-29-amazon-fsx-replaces-file-shares.md]: Gotcha, thanks

**Comment by mbonig** [comment]
K, I believe they're all done and ready.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-0e36980d22/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-0e36980d22/)

**Pull request merged**